db/migrate/20180410125416_rename_ansible_job_categories.rb

```diff
@@ -0,0 +1,17 @@
+class RenameAnsibleJobCategories < ActiveRecord::Migration[5.1]
+  def up
+    User.as_anonymous_admin do
+      updated_templates = ['Power Action - Ansible Default',
+                           'Puppet Run Once - Ansible Default']
```

Comment:
Looks like the `Service Action` template is also going to be updated

Reply:
that's handled by https://github.com/theforeman/foreman_ansible/pull/159/files#diff-fe9a9a343fbfce6ef20705c597e95427R13